Jenkins.build

```diff
@@ -1,4 +1,4 @@
-@Library(value='iow-ecs-pipeline@1.0.0', changelog=false) _
+@Library(value='iow-ecs-pipeline@single-registry', changelog=false) _
```

Comment:
I should have made this a proper draft PR.  These branch/tag references will change to something like 1.1.0 when changes from IOW-429 go live.